File: epaas/selling/report/sales_order_trends/sales_order_trends.js
// Copyright (c) 2015, Dataent Technologies Pvt. Ltd. and Contributors
// License: GNU General Public License v3. See license.txt

dataent.require("assets/epaas/js/sales_trends_filters.js", function() {
	dataent.query_reports["Sales Order Trends"] = {
		filters: epaas.get_sales_trends_filters()
	}
});

File: epaas/selling/report/sales_order_trends/sales_order_trends.py
# Copyright (c) 2015, Dataent Technologies Pvt. Ltd. and Contributors
# License: GNU General Public License v3. See license.txt

from __future__ import unicode_literals
import dataent
from epaas.controllers.trends	import get_columns,get_data

def execute(filters=None):
	if not filters: filters ={}
	data = []
	conditions = get_columns(filters, "Sales Order")
	data = get_data(filters, conditions)
	return conditions["columns"], data
